runbotics/runbotics-common/src/model/api/feature-key.model.ts

```diff
@@ -1,56 +1,149 @@
 export enum FeatureKey {
-    PROCESS_READ = 'PROCESS_READ', // lista procesow = /app/processes + konkretny proces
-    PROCESS_ADD = 'PROCESS_ADD', // przycisk dodawania
-    PROCESS_EDIT_INFO = 'PROCESS_EDIT_INFO', // akcja edit na kafelku
-    PROCESS_DELETE = 'PROCESS_DELETE', // akcja delete na kafelku
-    PROCESS_START = 'PROCESS_START', // ukryj przycisku RUN na procesie
-    PROCESS_EDIT_STRUCTURE = 'PROCESS_EDIT_STRUCTURE', //  tylko read to wyswietl viewer zamiast modelera, nie mozna wyswieltac panulu info actions
+    // Add switcher "display processes as a LIST or GRID" (by default it's grid) [ Processes page ]
+    PROCESS_LIST_TABLE_VIEW = 'PROCESS_LIST_TABLE_VIEW', 
+
+    // Show details on process tile (like: date created, author, time updated, is scheduled, etc.) [ Processes page ]
+    PROCESS_LIST_DETAIL_VIEW = 'PROCESS_LIST_DETAIL_VIEW', 
+
+    // Allow to GET single process by id [ Processes page -> process ]
+    PROCESS_READ = 'PROCESS_READ', 
+    
+    // Allow to GET list of all processes & GET single process by id [ Processes page ]
+    PROCESS_LIST_READ = 'PROCESS_LIST_READ',
+
+    // Show "ADD NEW PROCESS" button [ Processes page ]
+    PROCESS_ADD = 'PROCESS_ADD', 
+
+    // Action "Edit" on process tile -> 3 vertical dots (like: edit name, description, etc.) [ Processes page ]
+    PROCESS_EDIT_INFO = 'PROCESS_EDIT_INFO', 
+
+    // Action "Delete" on process tile -> 3 vertical dots [ Processes page ]
+    PROCESS_DELETE = 'PROCESS_DELETE', 
+
+    // Show "RUN" button & RUN tab [ Processes page -> process -> BUILD, RUN tabs ]
+    PROCESS_START = 'PROCESS_START', 
+
+    // Allow editing process structure in modeler (without it - only viewer mode & no access to panel info actions) [ Processes page -> process -> BUILD tab ]
+    PROCESS_EDIT_STRUCTURE = 'PROCESS_EDIT_STRUCTURE', 
+    
+    // ???
     PROCESS_IS_ATTENDED_EDIT = 'PROCESS_IS_ATTENDED_EDIT',
+
+    // ====== NEVER USED ======
```

Comment:
rly? it should be xd

Reply:
Looks like it's not. Tried "search" in vsc

Found out that external user with no access to "configure" tab can access it through url:
![image](https://user-images.githubusercontent.com/69820069/193035273-9ed87ed1-28c6-4a29-b299-64aa34e9c28e.png)

Looks like somebody who added these keys missed it. I've added logic which hides switchers "Is process triggerable" and "Is process attended".  Now, there're unused keys no more.